kofta/src/vscode-webview/Routes.tsx

```diff
@@ -393,6 +402,11 @@ export const Routes: React.FC<RoutesProps> = () => {
 				setPublicRooms(() => ({ publicRooms, nextCursor }));
 			},
 			join_room_done: (d) => {
+				// Auto open chat to show description and if mobile
+				if (shouldBeSidebar) {
+					useRoomChatStore.getState().toggleOpen();
```

Comment:
someone could have chat from previous room already open

Reply:
yeah, I tried that but seemed like once they leave a room, it is closed.